.github/workflows/tag.yaml

```diff
@@ -15,10 +15,12 @@ on:
   # and releasing too infrequently which delays delivery of bugfixes and features.
   schedule:
     - cron: "0 15 1,15 * *"
+permissions:
+  id-token: write
+  attestations: write
+  contents: write
```

Comment:
Does the tag workflow need these permissions because it kicks off the release workflow?

Reply:
Yes, that's my intent in putting these here, but I'm not sure they are actually required. I should remove the first two and test whether it works without them.